src/types.ts

```diff
@@ -86,7 +86,7 @@ export interface Options {
    *
    * @default 'vue3'
    */
-  transformer?: 'vue3' | 'vue2'
+  transformer?: string
```

Comment:
```suggestion
  transformer?: 'vue3' | 'vue2' | 'svelte3'
```

Reply:
Yeah i guess we could leave it as that for now, but if the goal is to allow custom transformers, it will need to become a string someday

Reply:
This way is friendly to typescript autocomplete